src/views/Project/index.jsx

```diff
@@ -120,7 +120,9 @@ export default class Project extends Component {
           </Typography>
           {data && data.error && <ErrorPanel error={data.error} />}
           {loading && <Spinner />}
-          {!loading && <TasksTable items={issues} />}
+          {!loading && (
+            <TasksTable history={history} location={location} items={issues} />
```

Comment:
Remove `location` and `history`. Use location directly in `TasksTable` and use `@withRouter` decorator to get access to history.